components/ukm/BUILD.gn

```diff
@@ -2,7 +2,9 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import("//testing/test.gni")
+if (!use_cobalt_customizations) {
+  import("//testing/test.gni")
```

Comment:
No action needed, but i still think it would be cleaner to just create a stub no-op file for this.

Reply:
ACK. Should probably do it all at once in a separate CL.